app/api/utils.py

```diff
@@ -455,6 +455,37 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
                              allow_nan=not self.strict) + '\n'
 
 
+class FastTextPainter(object):
+
+    @staticmethod
+    def paint_labels(documents, labels):
+        serializer = DocumentSerializer(documents, many=True)
+        serializer_labels = LabelSerializer(labels, many=True)
+        data = []
+        for d in serializer.data:
+            labels = []
+            for a in d['annotations']:
+                label_obj = [x for x in serializer_labels.data if x['id'] == a['label']][0]
+                labels.append('__label__{}'.format(label_obj['text'].replace(' ', '')))
+            data.append(' '.join(labels) + ' ' + d['text'].rstrip() if labels else d['text'].rstrip())
```

Comment:
newlines could be inside the text. replace them with a space. better to use str.format here, too.